ompp-run-ubuntu/Dockerfile

```diff
@@ -68,9 +68,9 @@ RUN GO_VER=1.22.5; \
   rm /tmp/go_setup.tar.gz
 
 # Install kubectl cli to be able to utilize file transfer functionality between containers
-ARG KUBECTL_VERSION=v1.28.2
+ARG KUBECTL_VERSION=v1.31.2
```

Comment:
Kubectl should be +/- 1 minor ver of k8s version (1.28.x)
I believe CNS wants to upgrade before Jan but idk if they'll make it.